backend-rust/src/migrations/m00010-ranking-by-total-stake.sql

```diff
@@ -0,0 +1,11 @@
+-- Rename table and colum since the table is re-created with new data for each block and not only for each payday block now.
+ALTER TABLE bakers_payday_lottery_powers 
+RENAME TO bakers_lottery_powers;
+ALTER TABLE bakers_lottery_powers
+RENAME COLUMN payday_lottery_power TO lottery_power;
```

Comment:
Unless we are okay with downtime, this renaming needs to be split into two migrations and two releases, one introducing the new table another removing the old (which is marked as destructive).
But since this information only changes at payday I think the naming is fine.